src/Fixer/Phpdoc/PhpdocLineSpanFixer.php

```diff
@@ -144,12 +144,24 @@ private function getDocBlockIndex(Tokens $tokens, int $index): int
             CT::T_NULLABLE_TYPE,
         ];
 
+        if (\defined('T_ATTRIBUTE')) { // @TODO: drop condition when PHP 8.0+ is required
+            $propertyPartKinds[] = T_ATTRIBUTE;
+        }
+
         if (\defined('T_READONLY')) { // @TODO: drop condition when PHP 8.1+ is required
             $propertyPartKinds[] = T_READONLY;
         }
 
         do {
             $index = $tokens->getPrevNonWhitespace($index);
+
+            if (!\defined('T_ATTRIBUTE')) { // @TODO: drop condition when PHP 8.0+ is required
```

Comment:
This `defined` check can be removed, there can never be a `CT::T_ATTRIBUTE_CLOSE` in the tokens if `T_ATTRIBUTE` is not defined.